packages/apps/desktop-viewer-test/src/frontend/components/signin/SignIn.tsx

```diff
@@ -0,0 +1,38 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import "./SignIn.scss";
+
+import { IModelApp } from "@bentley/imodeljs-frontend";
+import { Button } from "@itwin/itwinui-react";
+import React, { useState } from "react";
+
+export const SignIn = () => {
+  const [signingIn, setSigningIn] = useState(false);
+
+  const onSignInClick = () => {
+    setSigningIn(true);
+    IModelApp.authorizationClient?.signIn();
+  };
+
+  return (
+    <div className="signin-container">
+      <div className="signin">
+        <span className="icon icon-user signin-user" />
```

Comment:
nbd, but is there an icon from itwinui we can use? noit sure how many people will realize these icons come from the webfont

Reply:
yep. switched to an itwinui icon